qemu/tests/blockdev_commit_stress.py

```diff
@@ -0,0 +1,51 @@
+from virttest import utils_test
+
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitStress(BlockDevCommitTest):
+    def run_stress_test(self):
+        self.stress_test = utils_test.VMStress(self.main_vm, "stress",
+                                               self.params)
+        self.stress_test.load_stress_tool()
+
+    def stress_running_check(self):
+        if not self.stress_test.app_running():
+            raise self.test.error("Stress app does not running as expected")
```

Comment:
No need `raise`, just `self.test.error("Stress app does not running as expected")` is ok

Reply:
Have updated, thanks for pointing it out.